src/test/java/selenium/tutorial/studies/SingleClickNVerifyTest.java

```diff
@@ -1,4 +1,4 @@
-package com.mvn.Sample1;
+package selenium.tutorial.studies;

```

Comment:
Why this test class is written for?

Reply:
Why this test class is written for? If we are not using this test class, we can delete it.